src/functions/template-tags/date.php

```diff
@@ -14,117 +14,48 @@
-	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
-	 *
-	 * @return string|null Time
-	 */
-	function tribe_get_start_time( $event = null, $dateFormat = '', $timezone = null ) {
-		if ( is_null( $event ) ) {
-			global $post;
-			$event = $post;
-		}
-
-		if ( is_numeric( $event ) ) {
-			$event = get_post( $event );
-		}
-
-		if ( ! is_object( $event ) ) {
-			return;
-		}
-
-		if ( tribe_event_is_all_day( $event ) ) {
-			return;
-		}
-
-		$start_date = Tribe__Events__Timezones::event_start_timestamp( $event->ID, $timezone );
-
-		if ( '' == $dateFormat ) {
-			$dateFormat = tribe_get_time_format();
-		}
-
-		return tribe_format_date( $start_date, false, $dateFormat );
-	}
-}
-
-if ( ! function_exists( 'tribe_get_end_time' ) ) {
-	/**
-	 * End Time
-	 *
-	 * Returns the event end time
-	 *
-	 * @category Events
-	 * @see http://php.net/manual/en/function.date.php
-	 *
-	 * @param int    $event       (optional)
-	 * @param string $dateFormat  Allows date and time formating using standard php syntax
-	 * @param string $timezone    Timezone in which to present the date/time (or default behaviour if not set)
-	 *
-	 * @return string|null Time
-	 */
-	function tribe_get_end_time( $event = null, $dateFormat = '', $timezone = null ) {
-		if ( is_null( $event ) ) {
-			global $post;
-			$event = $post;
-		}
-
-		if ( is_numeric( $event ) ) {
-			$event = get_post( $event );
-		}
-
-		if ( ! is_object( $event ) ) {
-			return;
-		}
-
-		if ( tribe_event_is_all_day( $event ) ) {
-			return;
-		}
-
-		$end_date = Tribe__Events__Timezones::event_end_timestamp( $event->ID, $timezone );
-
-		if ( '' == $dateFormat ) {
-			$dateFormat = tribe_get_time_format();
-		}
-
-		return tribe_format_date( $end_date, false, $dateFormat );
-	}
-}
-
 if ( ! function_exists( 'tribe_get_display_end_date' ) ) {
 	/**
 	 * End Date formatted for display
 	 *
 	 * Returns the event end date that observes the end of day cutoff
 	 *
 	 * @category Events
-	 * @see http://php.net/manual/en/function.date.php
+	 * @see      http://php.net/manual/en/function.date.php
 	 *
-	 * @param int    $event        (optional)
-	 * @param bool   $display_time If true shows date and time, if false only shows date
-	 * @param string $date_format  Allows date and time formating using standard php syntax
-	 * @param string $timezone     Timezone in which to present the date/time (or default behaviour if not set)
+	 * @param int|WP_Post $event        The event (optional).
+	 * @param bool        $display_time If true shows date and time, if false only shows date.
+	 * @param string      $date_format  Allows date and time formatting using standard php syntax.
+	 * @param string      $timezone     Timezone in which to present the date/time (or default behaviour if not set).
 	 *
 	 * @return string|null Date
 	 */
 	function tribe_get_display_end_date( $event = null, $display_time = true, $date_format = '', $timezone = null ) {
```

Comment:
Hmm. This PR shouldn't be deprecating anything. It is removing some duplicate functions, those functions are already declared in Common and thus we don't need to redeclare them here. The ones in Common are already loaded before the ones here, and thus nothing is changing it's just removing orphaned code.

The other aspect of this PR is adding a little filter to `tribe_get_display_end_date()`. I was adding this filter simply because that function is missing one, yet most all of our template functions contain a filter. I was just trying to bring it inline with the rest. But, I could see an argument to be made for deprecating `tribe_get_display_end_date()`. And if that is something we're interested in doing, maybe adding a filter to it right now is a bad idea? 

Does that help clarify? Am I misunderstanding you?